rasa/utils/common.py

```diff
@@ -504,3 +504,14 @@ def find_unavailable_packages(package_names: List[Text]) -> Set[Text]:
 def module_path_from_class(clazz: Type) -> Text:
     """Return the module path of an instance's class."""
     return clazz.__module__ + "." + clazz.__name__
+
+
+def get_bool_env_variable(variable_name: str, default_variable_value: bool) -> bool:
+    true_values = ("true", "1")
+    false_values = ("false", "0")
```

Comment:
I would actually keep original boolean and int data types, rather than having this as string, avoids unnecessary type casting.

Reply:
Well these are the valid input values which are gonna be returned by the os.getenv.
So when env variable is set and we fetch it with os.getenv, we will get string representation like:
"False", "false", "1" etc, so this is why I opt out to use strings.
I can switch to use:
```python
    true_values = (str(True).lower(), str(1).lower())
    false_values = (str(False).lower(), str(0).lower())
```
which will prevent any typos.